compiler/CCT/fail/strict.jazz

```diff
@@ -1,4 +1,4 @@
-fn add(#public reg u64 p, #secret reg u64 s) -> reg u64 {
+export fn add(#public reg u64 p, #secret reg u64 s) -> reg u64 {
```

Comment:
It is not related to this PR. But I don't understand why this test should fail.

Reply:
Running `jazzct --infer` gives me:
```
"CCT/fail/strict.jazz", line 2 (3-4):
constant type checker: p has type #strict #public it cannot receive a value of type #secret
```

Reply:
Thank I forgot that by default type annotated variable
are strict.